updateRuntimeVersions.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# USAGE:
+# > RUNTIME_VERSION=1.0.31 ./updateRuntimeVersions.sh
```

Comment:
I'm not sure where this is called from during the Runtime release process. The file rename must be reflected there.

Reply:
I think this is only used by us (by me at least).

Reply:
Its used [here](https://github.com/lightbend/kalix-runtime/blob/9d45ce3742bf3de179c16ae08691a144167b4d23/.circleci/config.yml#L576C15-L576C34) when opening a Auto PR upon runtime release.